apis/v1alpha1/gateway_types.go

```diff
@@ -223,6 +215,7 @@ type HostnameMatch struct {
 	// This field is required for the "Domain" and "Exact" match types.
 	//
 	// +optional
+	// +kubebuilder:validation:MaxLength=253
 	Name string `json:"name"`
```

Comment:
Add `omitempty` tag if this field is optional.

Reply:
Updated, thanks!